DOCS-11819-mqtt3-connector-final-kt (#1611)
mulesoft/docs-connectors: mqtt 3/1.0.0/modules/ROOT/pages/mqtt3-connector-publish-receive.adoc

		</mqtt3:listener>
		<logger level="INFO" message="Received message '#[payload]' with at topic #[attributes.topic] with qos #[attributes.qos]"/>
	</flow>
----

[start=4]
. Configure a new `<mqtt3:config name="MQTT3_Config_2">` configuration:

[source,xml,linenums]
----
<mqtt3:config name="MQTT3_Config_2">
	<mqtt3:connection username="mosquitto" password="mosquitto" url="tcp://0.0.0.0:1884">
	    <mqtt3:client-id-generator>
            <mqtt3:client-id-random-suffix-generator clientId="azfell123" />
        </mqtt3:client-id-generator>
	</mqtt3:connection>
</mqtt3:config>
----

[start=5]
. Create a second listener flow by adding a listener source `<mqtt3:listener>` that subscribes to the topic `topic/lwt`, and add a `<logger>` component that logs the received message:

[source,xml,linenums]
----
<flow name="listener-lwt-flow">
	<mqtt3:listener doc:name="On New Message 2" config-ref="MQTT3_Config_2">
		<mqtt3:topics >
			<mqtt3:topic topicFilter="topic/lwt" />
		</mqtt3:topics>
	</mqtt3:listener>
	<logger level="INFO" message="Received message '#[payload]' with at topic #[attributes.topic] with qos #[attributes.qos]"/>
</flow>
----

[start=6]
. Save and run the Mule app.

> . Simulate the app crash by looking for the process ID of the Mule app and kill it in your terminal by using a command like the following: `kill -9 <process-id>`. +

Reviewer: ```suggestion
. To simulate an app crash, find the process ID of the Mule app and then kill the process in your terminal by using a command like the following: `kill -9 <process-id>`. +
```
